app/presenters/navigation/specialist_documents.rb

```diff
@@ -3,10 +3,13 @@ module SpecialistDocuments
     include Finders
 
     PARAMS_TO_IGNORE = %w(
+      assessment_date
```

Comment:
What happens if a finder adds a new type of date?
Don't facets have a type that could be checked?